### PR TITLE
Kiko/feature/top100

### PR DESCRIPTION
**Pull request by knpausch**
# Description
-This builds out top100 component with game cards. Each card includes: name, rank, image, price, and userRating. Basic css styling is included for readability. 

# Fixes # (issue)
-Removed index element route from App.tsx as it was dead code without functionality and prevented the display of top100 component

# Type of change

- [x]  Bug fix (non-breaking change which fixes an issue)
 
- [x]  New feature (non-breaking change which adds functionality)

- [ ]  Breaking change (fix or feature that would cause existing functionality not to work as expected)

- [ ] This change requires a documentation update

-  [x] Styling

- [ ] Refactor

- [ ] Testing

# Where were the changes?
App.tsx, Top100.tsx, GameCard.tsx, GameCard.css

# How Has This Been Tested?
Tested numerous times on the DOM/eternal API backend servers and using console logs to ensure it is still functioning as before and confirmed this is the case. Also, all new and old tests are passing.

- [x] Test Approved

# Checklist:

- [x]  My code follows the style guidelines of this project
- [x]  I have performed a self-review of my code
- [ ]  I have commented my code, particularly in hard-to-understand areas
- [ ]  I have made corresponding changes to the documentation
- [x]  My changes generate no new warnings
- [ ]  I have added tests that prove my fix is effective or that my feature works
- [ ]  New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by dgubko — APPROVED** [review]
Looks good, thank you for working on it.

File: src/utilities/utilities.ts [hunk 1]
```diff
@@ -32,4 +32,165 @@ export const cleanDetails = (game: {games: GameDetails[]}) => {
         image: details["image_url"],
     }
     return cleanDetail;
-}
\ No newline at end of file
+}
+
+export const getRandomWord = () => {
+    return words[Math.floor(Math.random() * words.length)]
+}
```
Comment (Nagel29) [inline src/utilities/utilities.ts:39]: Love the random word generator!

**Review by Nagel29 — COMMENTED** [review]
This looks great!

**Review by Courtney-Lippman — APPROVED** [review]
This looks great!

**Pull request merged**